macros/generate_model_yaml.sql

```diff
@@ -17,24 +17,34 @@
     {% do return(model_yaml) %}
 {% endmacro %}
 
-{% macro generate_model_yaml(model_name, upstream_descriptions=False) %}
+{% macro generate_model_yaml(model_name=[], upstream_descriptions=False) %}
 
-{% set model_yaml=[] %}
-{% set column_desc_dict =  codegen.build_dict_column_descriptions(model_name) if upstream_descriptions else {} %}
+    {% set model_yaml=[] %}
+    {% set column_desc_dict =  codegen.build_dict_column_descriptions(model_name) if upstream_descriptions else {} %}
 
-{% do model_yaml.append('version: 2') %}
-{% do model_yaml.append('') %}
-{% do model_yaml.append('models:') %}
-{% do model_yaml.append('  - name: ' ~ model_name | lower) %}
-{% do model_yaml.append('    description: ""') %}
-{% do model_yaml.append('    columns:') %}
+    {% do model_yaml.append('version: 2') %}
+    {% do model_yaml.append('') %}
+    {% do model_yaml.append('models:') %}
+
+    {% if model_name is string %}
+        {% set model_name_list=[] %}
+        {% do model_name_list.append(model_name) %}
```

Comment:
```suggestion
        {% set model_name_list=[model_name] %}
```
I think you can do this in one line! (This becomes redundant if you accept my other comment, but is still a fun fact)

Reply:
good to know!